pom.xml

```diff
@@ -44,7 +44,7 @@
     </dependencies>
 
     <properties>
-        <oap.project.version>17.10.31.10</oap.project.version>
+        <oap.project.version>17.10.32.10</oap.project.version>
```

Comment:
Just a note: need to plan in separate task updating `OAP` (within parent POM) to 18 version

Reply:
Separate task:
[OAP-98](https://youtrack.xenoss.io/issue/OAP-98) Update Parent POM and migrate to JDK18 for OAP projects